### PR TITLE
Template Based Query Generation: Query generation (Merge Fix)

### PR DESCRIPTION
**Pull request by spoiledhua**
-This PR just fixes a merge error that occurred in the previous PR.

### CODE REVIEW

File: tools/template_based_query_generation/src/main/java/QueryGenerator.java [hunk 1]
```diff
@@ -1,31 +1,146 @@
...
+import java.util.*;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
 
 /**
  * Class that parses config file and creates queries from markov chain
  */
 public class QueryGenerator {
 
+	private final String filePathConfigDDL = "./src/main/resources/user_config/ddl.json";
```
Comment (Bei95) [inline tools/template_based_query_generation/src/main/java/QueryGenerator.java:23]: Please fix the indentation of this file to use 2 space of tab

File: tools/template_based_query_generation/src/main/java/QueryGenerator.java [hunk 2]
```diff
@@ -1,31 +1,146 @@
...
+	private final String filePathConfigDML = "./src/main/resources/user_config/dml.json";
+	private final String filePathConfigDQL = "./src/main/resources/user_config/dql.json";
+	private final String filePathDependenciesRoot = "./src/main/resources/dialect_config/root_dependencies.json";
+	private final String filePathDependenciesDDL = "./src/main/resources/dialect_config/ddl_dependencies.json";
```
Comment (Bei95) [inline tools/template_based_query_generation/src/main/java/QueryGenerator.java:27]: As discussed offline, please consider generalize this file to easier add configs for other dialects.

File: tools/template_based_query_generation/src/main/java/parser/Pair.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package parser;
+
+public class Pair<F, S> {
```
Comment (Bei95) [inline tools/template_based_query_generation/src/main/java/parser/Pair.java:3]: Java has it's own pair class. Any reason to create another pair class here?
https://docs.oracle.com/javase/8/javafx/api/javafx/util/Pair.html

File: tools/template_based_query_generation/src/main/java/QueryGenerator.java (hunk 1 above)
Comment (AllenWang314) [reply tools/template_based_query_generation/src/main/java/QueryGenerator.java:23]: Resolved!

File: tools/template_based_query_generation/src/main/java/parser/Pair.java (hunk 3 above)
Comment (AllenWang314) [reply tools/template_based_query_generation/src/main/java/parser/Pair.java:3]: For some reason, Intellij + maven is giving me difficulties with javafx (this was why I decided to do it myself originally). I have deleted the class and used import internal.net.http.common.Pair instead

Comment (AllenWang314) [reply tools/template_based_query_generation/src/main/java/parser/Pair.java]: So turns out that package isn't that great to work with either. This one was the one that worked in the end: org.apache.commons.lang3.tuple.MutablePair

**Pull request merged**